django_camunda/dmn/utils.py

```diff
@@ -1,9 +1,11 @@
 from datetime import datetime
-from typing import Tuple
+from typing import List, Tuple
```

Comment:
This library supports python 3.8

Reply:
I'm okay with dropping support for anything older than 3.10, let's tackle that at another time :)